api/pom.xml

```diff
@@ -34,6 +34,12 @@
             <version>${osgi-annotation.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.annotation.versioning</artifactId>
+            <version>1.1.0</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
This is already present in the same `<dependencies>` section and becomes redundant, please remove it.

```suggestion
```